lacework/resource_lacework_integration_aws_cfg.go

```diff
@@ -34,6 +35,11 @@ func resourceLaceworkIntegrationAwsCfg() *schema.Resource {
 				Optional: true,
 				Default:  true,
 			},
+			"retries": {
```

Comment:
@scottford-lw @dmurray-lacework Should this be `max_retries` instead? 🤔

Reply:
Yes, i think that better describes this parameters function.

Reply:
`max_retries` seems to be more the common nomenclature. At least from a quick search.
https://registry.terraform.io/providers/hashicorp/aws/latest/docs#max_retries

Not a sticking point for me and happy for this to go with either. But I do prefer max_retries